dev/raphael.core.js

```diff
@@ -339,7 +339,8 @@
         b = d.firstChild;
         b.style.behavior = "url(#default#VML)";
         if (!(b && typeof b.adj == "object")) {
-            return (R.type = E);
+            R.type = E;
+            return R;
```

Comment:
Oops! Now I see you meant to return R rather than E, which represents R.type.  :P